components/com_finder/tmpl/search/default_result.php

```diff
@@ -72,9 +72,9 @@
 }
 ?>
 <li class="result__item">
-	<?php if (isset($image)) : ?>
+	<?php if (isset($this->result->imageUrl)) : ?>
```

Comment:
```suggestion
	<?php if ($showImage && isset($this->result->imageUrl)) : ?>
```
This should fix the issue that images are also shown when the option to show them is switched off.